ci/change-runtime-settings.py

```diff
@@ -0,0 +1,72 @@
+# coding=utf-8
+
+# Purpose:
+#   change runtime settings for build android and ios package
+
+import os
+import sys
+import configparser
+from collections import OrderedDict
+
+
+class MultiOrderedDict(OrderedDict):
+    def __setitem__(self, key, value):
+        if isinstance(value, list) and key in self and key.startswith('+'):
+            self[key].extend(value)
+        else:
+            super(MultiOrderedDict, self).__setitem__(key, value)
+
+class CustomInterpolation(configparser.BasicInterpolation):
+    def before_write(self, parser, section, option, value):
+        if option.startswith('+'):
+            return "\n{}=".format(option).join(value.split('\n'))
+        return value
+
+class CustomWriter:
+    output_file = None
+    def __init__(self, new_output_file):
+        self.output_file = new_output_file
+    def write(self, what):
+        self.output_file.write(what.replace('\n\t','\n'))
+        
+
+# modify runtime settings before cook
+def change_runtime_settings(project_home):
+    defaultengine = os.path.join(project_home, 'Game', 'Config', 'DefaultEngine.ini')
+    config = configparser.ConfigParser(strict=False, dict_type=MultiOrderedDict, interpolation=CustomInterpolation())
+    config.optionxform = lambda option: option  # preserve case for letters
+    config.read(defaultengine)
+
+    IOSRuntimeSettings = '/Script/IOSRuntimeSettings.IOSRuntimeSettings'
+    if not config.has_section(IOSRuntimeSettings):
+        config.add_section(IOSRuntimeSettings)
+    
+    # for ios runtime settings
+    AdditionalPlistData = '<key>CFBundleURLTypes</key><array><dict><key>CFBundleURLName</key><string></string><key>CFBundleTypeRole</key><string>Editor</string><key>CFBundleURLSchemes</key><array><string>firebase-game-loop</string></array></dict></array>'
+    config[IOSRuntimeSettings]['AdditionalPlistData'] = AdditionalPlistData
+    config[IOSRuntimeSettings]['BundleIdentifier'] = 'io.improbable.unrealgdkdemo'
+    config[IOSRuntimeSettings]['MobileProvision'] = ''
+    config[IOSRuntimeSettings]['BundleDisplayName'] = 'UnrealGDK Shooter'
+    config[IOSRuntimeSettings]['BundleName'] = 'unrealgdkshooter'
```

Comment:
```suggestion
    config[IOSRuntimeSettings]['BundleName'] = 'unrealgdkexampleproject'
```
How come this one is lower case?